packages/base-driver/test/basedriver/driver-tests.js

```diff
@@ -335,8 +335,8 @@ function baseDriverUnitTests (DriverClass, defaultCaps = {}) {
         it('should have a #canProxy method', function () {
           d.canProxy.should.be.an.instanceof(Function);
         });
-        it('should return false from #canProxy', function () {
-          d.canProxy(sessId).should.be.false;
+        it('should return a boolean from #canProxy', function () {
+          [true, false].should.include(d.canProxy(sessId));
```

Comment:
I'd probably write this as

```suggestion
          d.canProxy(sessId).should.be.a('boolean');
```

but that's just me

Reply:
that's what I tried to do, with `.should.be.a(Boolean)`, but it broke and I didn't know there was a better way!